tools/twix/src/panels/plot.rs

```diff
@@ -164,10 +235,27 @@ impl Panel for PlotPanel {
 
 impl PlotPanel {
     fn plot(&self, ui: &mut Ui) -> Response {
+        let maximum_buffer_size = self
+            .line_datas
+            .iter()
+            .filter_map(|line_data| {
+                line_data
+                    .value_buffer
+                    .as_ref()
+                    .and_then(|buffer| buffer.get_buffer_size().ok())
+            })
```

Comment:
Didn't test it, but:
```suggestion
              .filter_map(|line_data| {
                let buffer = line_data.value_buffer.as_ref()?;
                buffer.get_buffer_size().ok()
            })

```

Reply:
Works fine, thanks!